server/schedule/checker/rule_checker.go

```diff
@@ -101,25 +123,44 @@ func (c *RuleChecker) fixRange(region *core.RegionInfo) *operator.Operator {
 	return op
 }
 
-func (c *RuleChecker) fixRulePeer(region *core.RegionInfo, fit *placement.RegionFit, rf *placement.RuleFit) (*operator.Operator, error) {
+// fixRulePeer fix region to satisfy rule requirement and statics miss region count include down and offline peer
+func (c *RuleChecker) fixRulePeer(region *core.RegionInfo, fit *placement.RegionFit, rf *placement.RuleFit) (op *operator.Operator, err error) {
 	// make up peers.
 	if len(rf.Peers) < rf.Rule.Count {
-		return c.addRulePeer(region, rf)
+		op, err = c.addRulePeer(region, rf)
```

Comment:
Will the operator be overwritten?

Reply:
no, it only statics the loss regions,do not generate operator.

Reply:
In the previous implementation, we generate operator here.